openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if requirements is None:
+    if not requirements:
```

Comment:
The code around here: https://github.com/edx/edx-ora2/blob/efischer/staff_endpoint/openassessment/workflow/models.py#L203 can send `{}`, which causes errors at line 55 below if we don't check for that case as well.

Would it be better to check for `None` and `{}` individually?